### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by chevdor**
### What does this PR do?

FIx minor typo and phrasing.

### CODE REVIEW

**Review by slemeur — APPROVED** [review]
Thank you for the PR @chevdor !

File: website/docs/troubleshooting.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ You can
 
 Then run a terminal in native mode (default) and install Podman machine `brew install podman-machine`
 
-Finally clean the poddman machine VMs that had been previously created, a create new ones.
+Finally clean the podman machine VMs that had been previously created and create new ones.
```
Comment (jeffmaury) [inline website/docs/troubleshooting.md]:
```suggestion
Finally clean the Podman machine VMs that had been previously created and create new ones.
```

Comment (themr0c) [inline website/docs/troubleshooting.md]:
```suggestion
Finally clean the Podman machine VMs that had been previously created and create new ones.
```

File: website/docs/troubleshooting.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ You can
 
 Then run a terminal in native mode (default) and install Podman machine `brew install podman-machine`
 
-Finally clean the poddman machine VMs that had been previously created, a create new ones.
+Finally clean the Podman machine VMs that had been previously created and create new ones.
```
Comment (themr0c) [inline website/docs/troubleshooting.md]:
```suggestion
Finally clean the Podman machine VMs that had been previously created, and create new ones.
```

**Review by themr0c — APPROVED** [review]

**Pull request merged**